### DISCUSSION TITLE
[FR]: Dynamic date filters

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#571] [Ideas]
### Present your request

Adding filters for dates that change dynamically with the day.

eg:
date = today
date = before today
date = after today
etc

### For which platform do you request this request??

Desktop

**Comment by RafaelGB** [comment]
you can achieve something similar if you associate a filter to the value of a formula with that behavior. You can leave it hidden if you don't want to show it

**Comment by DannyHatcher-Git** [comment]
I thought about doing that but think it would be easier for new users to have the option in the filter.

That is what I am doing right now.

**Comment by RafaelGB** [comment]
I think its too specific feature. Maybe in the future, but I think we have to move this to discussions

**Comment by kalmir** [comment]
I think everyone using db-folder for task management would welcome this feature. Myself included 🙂

**Comment by irongiantfm** [comment]
Kind of glad to find out this was a missing feature. I was driving myself insane on figuring out how to write this filter. Definitely a feature request for this guy.

**Reply by RafaelGB** [reply]
we can achieve this with dataview queries as source. While we find a way to configure it with another source

**Comment by irongiantfm** [comment]
Sounds good. Will do
